File: circlething.cpp
#include <iostream>
#include <cmath>
int main(){
    bool REPEAT = true;
    std::string UserChoice;
    while(REPEAT == true) {

        std::cout << "Would you like to run the program? (y/n)\n";
        std::cin >> UserChoice;

        if(UserChoice == "y") {
            const float Pi = 3.142;
            float UserRadius;
            std::cout << "What's the radius of the circle?\n";
            std::cin >> UserRadius;
            float CircleArea;
            std::cout << "The area is ";
            std::cout << pow(UserRadius,2) * Pi;
            std::cout << "\n";
        } else if (UserChoice == "n") {
            break;
        } else {       
        }
    }
}

File: circlething.py
Pi = float(3.142)
REPEAT = True
while REPEAT == True:
  Userchoice = input("Do you want to run the program (y/n) ")
  if Userchoice.lower() == "y":
    UserRadius = float(input("What's the radius of the circle?\n"))
    CircleArea = float(UserRadius*UserRadius*Pi)
    print("The area is ", str(CircleArea))
  elif Userchoice.lower() == "n":
    break
  else:
    print("\n")
